src/openrct2/network/ServerList.cpp

```diff
@@ -80,3 +233,170 @@ bool server_list_write(const std::vector<server_entry>& entries)
         return false;
     }
 }
+
+std::future<std::vector<ServerListEntry>> ServerList::FetchLocalServerListAsync(const INetworkEndpoint& broadcastEndpoint)
+{
+    auto broadcastAddress = broadcastEndpoint.GetHostname();
+    return std::async([broadcastAddress] {
```

Comment:
Perhaps use std::launch::async

Reply:
Is there any difference? I couldn't see one during testing.